internal/pkg/aws/cloudwatch/cloudwatch_test.go

```diff
@@ -353,3 +354,159 @@ func TestCloudWatch_AlarmsWithTags(t *testing.T) {
 
 	}
 }
+
+func TestCloudWatch_AlarmStatusesFromNamePrefix(t *testing.T) {
```

Comment:
nit: since we are not adding a new exported func now, we can probably just add some of the unit test cases to `TestCloudWatch_AlarmStatuses` instead of having a new test suite!

Reply:
Good idea. There isn't currently a `TestCloudWatch_AlarmStatuses` test suite-- they're bundled into `AlarmsWithTags`, so I'll separate out and combine.